src/main/java/duke/function/UserGreeter.java

```diff
@@ -0,0 +1,36 @@
+package duke.function;
+
+public class UserGreeter {
```

Comment:
Good implementation of a class dedicated to printing out messages, making it easy to reference when we are looking for expected output